src/App.jsx

```diff
@@ -32,6 +36,10 @@ class App extends Component {
             <Route path="/articles/:slug" component={ViewArticle} />
             <Route path="/profile" component={ProfilePage} />
             <Route path="/social" component={SocialLogin} />
+            <Route path="/resetPassword" component={PasswordReset} />
```

Comment:
Hi @okoroemeka, awesome work putting this together. I see you named one of your route using camel case. I think it makes for better readability (a user standpoint) and SEO benefits to separate routes with more than one word with `-` character.

e.g. 
```js
/reset-password
```

Reply:
There may be better approaches, but please let me hear your thoughts on this.

Reply:
I completely agree with you Luqman on this. It also helps us maintain consistency on the project; since everyone has been using hyphens as word delimiter so far.

Reply:
I think this is a great idea @codeshifu .